src/firefly/java/edu/caltech/ipac/firefly/server/util/ipactable/JsonTableUtil.java

```diff
@@ -56,15 +56,17 @@ public static JSONObject toJsonTableModel(DataGroupPart page, TableServerRequest
             tableModel.put("tbl_id",  meta.getAttribute(TableServerRequest.TBL_ID).getValue());
         }
 
-        if (request.containsParam(ServerParams.USE_UPLOADED_FILENAME_AS_TABLE_TITLE) &&
+      /*  if (request.containsParam(ServerParams.USE_UPLOADED_FILENAME_AS_TABLE_TITLE) &&
                 request.getBooleanParam(ServerParams.USE_UPLOADED_FILENAME_AS_TABLE_TITLE)){
 
                 tableModel.put("title", page.getTableDef().getAttribute("title").getValue());
 
         }
         else {
             tableModel.put("title", page.getData().getTitle());
-        }
+        }*/
```

Comment:
The code was added by this PR.  Then it was deemed not necessary.  Why would you not removed it, but instead leaving it commented out?  How would this block of code ever get cleaned up?